cmd/eulabia-sensor/main.go

```diff
@@ -27,9 +28,11 @@ func main() {
 	c, err := mqtt.New(*server, *clientid+uuid.NewString(), "", "",
 		&mqtt.LastWillMessage{
 			Topic: topic,
-			MSG: messages.Delete{
-				ID:      *sensorID,
-				Message: messages.NewMessage("delete.sensor", "", ""),
+			MSG: cmds.Delete{
+				Identifier: messages.Identifier{
+					Message: messages.NewMessage("delete.sensor", "", ""),
+					ID:      *sensorID,
+				},
```

Comment:
The last will message should be sent to all Clients and to all sensors. So we need a topic that both of them should subscribe to, to detect that the director lost its connection.

Reply:
Yes and no; yes they're interested in but in the case we want the director to remove the sensor as a valid choice.

Reply:
So I think for that they should listen to:
`eulabeia/sensor/info/director` specifically `deleted.sensor`